test/Microsoft.Health.Fhir.Shared.Tests.Integration/TestScoped.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Extensions.DependencyInjection;
+
+namespace Microsoft.Health.Fhir.Shared.Tests.Integration
+{
+    public class TestScoped<T> : IScoped<T>
+    {
+        public T Value { get; set; }
```

Comment:
There is an existing helper method `CreateMockedScopeExtensions.CreateMockScope` that we use for other tests. Would it be possible to use that for the `ReindexJobTests` instead of adding a new class?